pkg/crc/oc/oc_cache.go

```diff
@@ -0,0 +1,170 @@
+package oc
+
+import (
+	"fmt"
```

Comment:
is this file formatted? looks like it does not use tab, but a spaced indent

Reply:
I did use `gofmt`, will check again.

Reply:
@gbraad I opened this file on vim and even goLand and both place I can see it as using tab not space, how did you open it?

Reply:
github interface